docs/data/material/migration/migration-v4/migration-v4.md

```diff
@@ -83,6 +83,20 @@ We will not change the minimum supported version in a major version of Material
 However, we generally recommend not to use a TypeScript version older than the lowest supported version of DefinitelyTyped.
 :::
```

Comment:
It feels like this part should be grouped with the one below that mentions `@types/react`. They are both about the same things: types.

Reply:
Actually, I think that it could be great to have a h3 React, and h3 TypeScript, to move `The minimum supported version of TypeScript has been increased from v3.2 to v3.5.` under the 2nd h3.